File: 01.ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Lab/src/P01_Passed.java
import java.util.Scanner;
public class P01_Passed {
    public static void main(String[] args) {

        Scanner console = new Scanner(System.in);
        double grade = Double.parseDouble(console.nextLine());
        System.out.println((grade >= 3.0) ? "Passed!":"");
    }
}

File: 01.ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Lab/P01_Passed/P01_Passed.cs
using System;

namespace P01_Passed
{
    class P01_Passed
    {
        static void Main(string[] args)
        {
            double grade = double.Parse(Console.ReadLine());
            Console.WriteLine((grade >= 3.0) ? "Passed!":"");
        }
    }
}
